packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraCaptureModeTests.m

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera_avfoundation;
+@import camera_avfoundation.Test;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+#import "CameraTestUtils.h"
+#import "MockFLTThreadSafeFlutterResult.h"
+
+@interface CameraCaptureModeTests : XCTestCase
+@property(readonly, nonatomic) FLTCam *camera;
+@end
+
+@implementation CameraCaptureModeTests
+
+- (void)setUp {
+  dispatch_queue_t captureSessionQueue = dispatch_queue_create("capture_session_queue", NULL);
+  dispatch_queue_set_specific(captureSessionQueue, FLTCaptureSessionQueueSpecific,
+                              (void *)FLTCaptureSessionQueueSpecific, NULL);
+
+  _camera = FLTCreateCamWithCaptureSessionQueue(captureSessionQueue);
```

Comment:
i would just inline this setup in the test, to avoid potential side effect of FLTCam